### DISCUSSION TITLE
Handlebars template styling is impacted by other web parts on page

### DISCUSSION THREAD

**Discussion by webbrewer** [microsoft-search/pnp-modern-search#1831] [Q&A]
v3.14.2
Search results web part with handlebars custom template.
If I add a highlighted content web part to the same page, the styling such as fonts follows the styling of the hcwp. Very odd.

**Comment by wobba** [comment]
Hard to say what causes, but you can always change styling in the template. Or use v4 which has improved isolation over v3 if I remember correctly.
